modules/rust_examples/RsExampleUser/src/eventually_generated.rs

```diff
@@ -1,18 +1,14 @@
+use everestrs::{Publisher, Subscriber};
 use std::collections::HashMap;
+use std::sync::Arc;
 
-/// This interface defines an example_user interface that uses the example interface
-pub trait ExampleUserService {}
-
-pub trait ExampleSubscriber {
-    fn on_max_current(&self, value: f64);
-}
-
-/// This interface defines an example interface that uses multiple framework features
-pub struct ExampleClient {
-    raw_publisher: everestrs::RawPublisher,
+/// The publisher for the example module.
+pub struct ExamplePublisher<'a> {
```

Comment:
The publishers are ligthweigth wrappers around the module - we will construct them a lot on the fly...

Reply:
Here is one important piece missing - The publisher should provide a handle to a struct allowing to get all publishers - otherwise there is no way to call `another_example` in the callback of the `their_example` handle